src/output/print.rs

```diff
@@ -9,7 +9,7 @@ pub trait Print {
 
 impl Print for &str {
     fn print(&self, writter: &mut dyn WriteColor) -> Result<()> {
-        write!(writter, "{}", self).with_context(|| {
+        write!(writter, "{}\n", self).with_context(|| {
```

Comment:
There is a macro `writeln!` that is adding a new line for you, I would advise to use it instead.

Reply:
The new PR is ready.
https://github.com/soywod/himalaya/pull/251